### PR TITLE
removal of superfluous comments 

### PR DESCRIPTION
**Pull request by LachlannRattray**
removed some un-needed code comments
re-factored empty else if statements

### CODE REVIEW

**Comment by emmaburstow** [comment]
Oh hi @LachlannRattray 

I'll have a look over this and let you know if it breaks anything ;)

Emma, [Community Pull Request team](https://umbraco.com/blog/introducing-the-new-pr-team/)

File: src/Umbraco.Web.UI.Client/src/common/directives/components/tree/umbtree.directive.js [hunk 1]
```diff
@@ -1,422 +1,422 @@
...
-            customtreeparams: '@',
-            eventhandler: '=',
-            enablecheckboxes: '@',
-            enablelistviewsearch: '@',
-            enablelistviewexpand: '@'
-        },
-
-        compile: function (element, attrs) {
-            //config
-            //var showheader = (attrs.showheader !== 'false');
```
[the remark this answers is not in the document]
Comment (LachlannRattray) [reply src/Umbraco.Web.UI.Client/src/common/directives/components/tree/umbtree.directive.js]: Hey Sebastiaan sorry for the delay in getting back, I am looking into these. Weirdly these two files give me issues I am not even able to revert them in gitkraken as they automatically get reset with the new line endings. Every weird :( I will keep looking

**Comment by abjerner** [comment]
@LachlannRattray just checking up on this PR - are you able to make the changes suggested by @nul800sebastiaan?

**Comment by nul800sebastiaan** [comment]
Hey there @LachlannRattray !

I'm writing today after announcing at [Codegarden](https://codegarden19.com/), our yearly conference, that the upcoming version 7.15.0 will be the last Umbraco v7 release with new features in it. From now on all our efforts will be focused on version 8 instead. 🎱

We're wrapping up the 7.15 version at the moment and unfortunately these proposed changes won't be able to make it in there. 

We'd like to extend a big thank you the efforts on this so far and we would love it if you could have a look at porting this feature over to v8 instead. Of course if you have any questions on where to start then feel free to [ask on the forum](https://our.umbraco.com/forum/contributing-to-umbraco-cms/), we monitor all questions there and we're happy to assist.

Thanks again for the work in this PR and we hope to see you contributing to v8 some time soon!

Best, Sebastiaan and the Umbraco CMS team.

**Comment by LachlannRattray** [comment]
Hey Sebastiaan

Thanks for your email and sorry for not being able to follow up this PR. V8 is looking awesome and we are really enjoying working with it.

Cheers
L

**Pull request closed without merging**